plugins/modules/grafana_organization_user.py

```diff
@@ -0,0 +1,259 @@
+options:
+  login:
+    type: str
+    required: True
+    description:
+      - Username or email.
+  role:
+    type: str
+    choices:
+      - viewer
+      - editor
+      - admin
+    default: viewer
+    description:
+      - User's role in the organization.
+  state:
+    type: str
+    default: present
+    choices:
+      - present
+      - absent
+    description:
+      - Status of a user's organization membership.
+  org_id:
+    type: int
+    default: 1
+    description:
+      - Organization ID.
+
+extends_documentation_fragment:
+  - community.grafana.basic_auth
+'''
+
+EXAMPLES = '''
+---
+- name: Add user to organization
+  community.grafana.grafana_organization_user:
+    url: "{{ grafana_url }}"
+    url_username: "{{ grafana_username }}"
+    url_password: "{{ grafana_password }}"
+    login: john
+    role: admin
+
+- name: Remove user from organization
+  community.grafana.grafana_organization_user:
+    url: "{{ grafana_url }}"
+    url_username: "{{ grafana_username }}"
+    url_password: "{{ grafana_password }}"
+    login: john
+    state: absent
+'''
+
+RETURN = '''
+---
+user:
+    description: Information about the organization user
+    returned: when state present
+    type: complex
+    contains:
+        email:
+            description: The User email address
+            returned: always
+            type: str
+            sample:
+                - "foo.bar@example.com"
+        login:
+            description: The User login
+            returned: always
+            type: str
+            sample:
+                - "batman"
+        name:
+            description: The User name (same as login)
+            returned: always
+            type: str
+            sample:
+                - "batman"
+        orgId:
+            description: The organization id that the team is part of.
+            returned: always
+            type: int
+            sample:
+                - 1
+        role:
+            description: The user role in the organization
+            returned: always
+            type: str
+            choices:
+                - Viewer
+                - Editor
+                - Admin
+            sample:
+              - Viewer
+'''
+
+
+import json
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import fetch_url
+from ansible.module_utils._text import to_text
+from ansible_collections.community.grafana.plugins.module_utils.base import grafana_argument_spec, clean_url
+from ansible.module_utils.urls import basic_auth_header
+
+__metaclass__ = type
+
+
+class GrafanaAPIException(Exception):
+    pass
+
+
+class GrafanaOrganizationUserInterface(object):
+
+    def __init__(self, module):
+        self._module = module
+        # {{{ Authentication header
+        self.headers = {"Content-Type": "application/json"}
+        self.headers["Authorization"] = basic_auth_header(module.params['url_username'], module.params['url_password'])
```

Comment:
maybe you can add grafana_api_key usage (see https://github.com/ansible-collections/community.grafana/blob/main/plugins/modules/grafana_datasource.py#L673) for example.

Reply:
@seuf the org api endpoint used do not support api token auth, see https://grafana.com/docs/grafana/latest/developers/http_api/org/#admin-organizations-api

It was the same for the users module